packages/menu/src/Menu.ts

```diff
@@ -179,6 +191,10 @@ export class Menu extends SpectrumElement {
         focusInItem.tabIndex = 0;
     };
 
+    private isDirectChild(menuItem: Element): boolean {
```

Comment:
Not sure on best naming here; `isDirectChild` is slightly misleading since [Element.closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) includes self.